src/react-apps/applications/ux-editor/src/actions/formDesignerActions/actions/addApplicationMetadata/index.ts

```diff
@@ -0,0 +1,44 @@
+import { Action } from 'redux';
+import * as ActionTypes from '../../formDesignerActionTypes';
+
+export interface IAddApplicationMetadataAction extends Action {
+  id: string;
+  maxFiles: number;
+  maxSize: number;
+  fileType: string;
+  callback?: (...args: any[]) => any;
+}
+
+export interface IAddApplicationMetadataActionRejected extends Action {
+  error: Error;
+}
+
+export function addApplicationMetadataAction(
+  id: string,
+  maxFiles: number,
+  maxSize: number,
+  fileType: string,
+  callback?: (...args: any[]) => any): IAddApplicationMetadataAction {
+  return {
+    type: ActionTypes.ADD_APPLICATION_METADATA,
+    id,
+    maxFiles,
+    maxSize,
+    fileType,
+    callback,
```

Comment:
This callback can be removed from interface

Reply:
fixed